tests/fuzz/corpora/basewords.txt

```diff
@@ -0,0 +1,1468 @@
+tiers
+prise
+professionnels
+raisons
+néanmoins
+preuve
+social
+bénéficiaire
+couleurs
+mondial
+Cet
+maintenant
+essentiellement
+prévu
+Japon
+prévisions
+centrale
+Alors
+international
+yeux
+PME
+l'a
+ait
+bonnes
+opérations
+pied
+l'art
+pourraient
+Londres
+juge
+devra
+uniquement
+corps
+divers
+Parmi
+numéro
+réduire
+Tous
+texte
+tenu
+budget
+l'étranger
+pression
+mes
+n'était
+style
+économiques
+Jacques
+montre
+population
+analystes
+S
+processus
+placement
+classique
+dividende
+rester
+publics
+fortement
+plein
+wallonne
+DEM
+Express
+faudra
+travailler
+Crédit
+directement
+prime
+Flandre
+crédit
+monnaie
+précise
+appel
+Autre
+travaux
+l'occasion
+juste
+Chaque
+put
+tableau
+terre
+permettent
+devenu
+rouge
+mémoire
+partenaires
+rapide
+travailleurs
+joue
+objectif
+salle
+parle
+musique
+milieu
+d'entreprise
+autorités
+chute
+régime
+d'autant
+liste
+opération
+bout
+performances
+électronique
+haute
+responsables
+lancé
+voitures
+patron
+Malgré
+affiche
+situe
+B
+l'image
+études
+Microsoft
+condition
+retrouve
+Aux
+revient
+Belgacom
+route
+Ensuite
+Luxembourg
+campagne
+comptes
+hors
+culture
+Commission
+d'entre
+possibilités
+semestre
+actifs
+finalement
+internationale
+l'achat
+monétaire
+passage
+justice
+page
+tels
+poids
+celle-ci
+commercial
+entendu
+l'investisseur
+mondiale
+accord
+diverses
+totalement
+fil
+clair
+vin
+biens
+euro
+parfaitement
+viennent
+réseaux
+principal
+lancer
+supérieur
+atteindre
+référence
+téléphone
+management
+vins
+proche
+collection
+fiscale
+Ceci
+informatique
+investissement
+volume
+matériel
+publicité
+coupon
+progression
+tenir
+protection
+l'aide
+couleur
+nouvel
+Lorsque
+change
+changement
+garantie
+somme
+Belge
+como
+su
+él
+era
+para
+con
+ellos
+ser
+uno
+tener
+este
+desde
+por
+caliente
+palabra
+pero
+qué
+algunos
+lo
+usted
+o
+tenido
+nos
+lata
+fuera
+otros
+eran
+hacer
+tiempo
+lo hará
+cómo
+dicho
+cada
+decir
+hace
+conjunto
+tres
+querer
+aire
+así
+también
+jugar
+pequeño
+fin
+poner
+casa
+leer
+mano
+puerto
+grande
+deletrear
+añadir
+incluso
+tierra
+aquí
+debe
+alto
+tal
+siga
+acto
+por qué
+preguntar
+hombres
+cambio
+se fue
+luz
+tipo
+necesitará
+imagen
+tratar
+nosotros
+de nuevo
+animal
+punto
+madre
+mundo
+cerca
+construir
+auto
+padre
+adivinar
+necesario
+agudo
+ala
+crear
+vecino
+lavado
+bate
+más bien
+multitud
+maíz
+comparar
+poema
+cadena
+campana
+dependerá
+carne
+frotar
+tubo
+famoso
+dólar
+corriente
+miedo
+vista
+delgado
+triángulo
+planeta
+prisa
+jefe
+colonia
+reloj
+mina
+empate
+entrar
+importante
+fresco
+búsqueda
+enviar
+amarillo
+pistola
+permitir
+print
+muerto
+lugar
+desierto
+traje
+actual
+ascensor
+rosa
+llegar
+master
+pista
+orilla
+división
+hoja
+sustancia
+favorecer
+conectar
+mensaje
+pasar
+acorde
+grasa
+contento
+originales
+cuota
+estación
+papá
+pan
+cobrar
+adecuada
+barra
+oferta
+segmento
+esclavo
+pato
+instantánea
+mercado
+grado
+poblar
+polluelo
+querido
+enemigo
+responder
+bebida
+producirse
```

Comment:
__[Misspell]__

"producirse" is a misspelling of "producers"


<!-- Comment by [Sider](https://sider.review) -->